### PR TITLE
Fixing plugins exposing for 2.x.

### PR DESCRIPTION
**Pull request by RoySegall**
#757

### CODE REVIEW

File: restful.module [hunk 1]
```diff
@@ -63,7 +63,17 @@ function restful_menu() {
       $items[$base_path . '/' . $plugin_definition['resource']] = $item;
     }
     else {
-      $path = implode('/', array($base_path, $plugin_definition['menuItem']));
```
Comment (e0ipso) [inline restful.module:66]: How about:

``` php
$path = implode('/', array($base_path, $plugin_definition['menuItem']));
// Remove trailing slashes that can lead to 404 errors.
$path = rtrim($path, '/');
```

**Comment by e0ipso** [comment]
You even got a test! I miss you around here @RoySegall!

**Comment by RoySegall** [comment]
This is how we roll in gizra :)

**Comment by RoySegall** [comment]
Improve. Ready to go.

> I miss you around here @RoySegall!

Good to be back ;)

**Comment by e0ipso** [comment]
Merged! Thanks.

**Pull request closed without merging**